jax/util.py

```diff
@@ -219,3 +219,9 @@ def get_module_functions(module):
         attr, (types.BuiltinFunctionType, types.FunctionType, onp.ufunc)):
       module_fns.add(attr)
   return module_fns
+
+def subvals(lst, replace):
```

Comment:
Please document.

(I know this moved from `lax.py`, but I didn't know what it did there either and I'm still hazy about what the name means.)

Reply:
Done.